GPU/Common/FramebufferCommon.cpp

```diff
@@ -575,7 +572,19 @@ void FramebufferManagerCommon::NotifyRenderFramebufferSwitched(VirtualFramebuffe
 
 	if (useBufferedRendering_) {
 		if (vfb->fbo) {
-			draw_->BindFramebufferAsRenderTarget(vfb->fbo);
+			if (gl_extensions.IsGLES) {
+				// Some tiled mobile GPUs benefit IMMENSELY from clearing an FBO before rendering
+				// to it. This broke stuff before, so now it only clears on the first use of an
+				// FBO in a frame. This means that some games won't be able to avoid the on-some-GPUs
+				// performance-crushing framebuffer reloads from RAM, but we'll have to live with that.
+				if (vfb->last_frame_render != gpuStats.numFlips) {
```

Comment:
Yeah, just moved it (while changing to the new call signature of BindFramebufferAsRenderTarget), so I'm gonna leave this for later...